package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "importer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "openhimConfig.js",
+  "scripts": {
+    "start": "node configBase.js",
+    "build": "esbuild configBase.js --bundle --platform=node --target=node10.4 --outfile=config.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "axios": "^0.26.1",
+    "esbuild": "^0.14.36",
+    "form-data": "^4.0.0"
```

Comment:
I'd recommend locking these dependencies to a specific version by removing ^'s

Reply:
Yep, it's done in [8796dbf](https://github.com/jembi/api-config-importer/pull/1/commits/8796dbfa02493f88ff15ce5cc85a93ec4735a0a7)